Update organizations documentation in the server admin guide (#33311)
keycloak/keycloak: docs/documentation/server_admin/topics/organizations/managing-identity-providers.adoc

Domain::
The domain from the organization that you want to link with the identity provider.

Hide on login page::
If this identity provider should be hidden in login pages when the user is authenticating in the scope of the organization.

Redirect when email domain matches::
If members should be automatically redirected to the identity provider when their email domain matches the domain set to the identity provider.

Once linked to an organization, the identity provider can be managed just like any other in a realm by accessing the *Identity Providers* section in the menu. However, the options herein described are only available when managing the identity provider in the scope of an organization. The only exception is the
 *Hide on login page* option that is present here for convenience.

== Editing a linked identity provider

> You can edit any of the organization-related settings of a linked identity provider at any time. go to the *Identity providers* tab in the organization, locate the provider in the list, then click the action button (with three dots) at the end of the line and select *Edit*.
> 

Reviewer: ```suggestion
You can edit any of the organization-related settings of a linked identity provider at any time. 
```

Reviewer: The procedure already has the steps.

.Procedure

> . Go to the *Identity providers* tab in the organization settings
> . Locate the *identity provider* in the list (you can use the search capabilities for that)
> . Click on the action button (three dots) at the end of the line
> . Click *Edit*
> . Make the necessary changes
> . Click *Save*

Reviewer: ```suggestion
. In the organization settings, go to the *Identity providers* tab.
. Locate the *identity provider* in the list.
+
You can use the search option for this step.
. Click the action button (three dots) at the end of the line.
. Click *Edit*.
. Make the necessary changes.
. Click *Save*.
```

Reviewer: We include a period for a complete sentence and use "click" not "click on"

.Editing linked identity provider
image:images/organizations-edit-identity-provider.png[alt="Editing linked identity provider"]

== Unlinking an identity provider from an organization

> When an identity provider is unlinked from an organization, it continues to be available as a realm-level provider that is not associated with an organization anymore. If you want to delete the unlinked provider, it has to be done through the *Identity Providers* section in the menu.
> 
> .Procedure
> 

Reviewer: ```suggestion
When an identity provider is unlinked from an organization, it remains available as a realm-level provider that is no longer ssociated with an organization. If you want to delete the unlinked provider, use the *Identity Providers* section in the menu.

.Procedure

```

> . Go to the *Identity providers* tab in the organization settings
> . Locate the *identity provider* in the list (you can use the search capabilities for that)
> . Click on the action button (three dots) at the end of the line
> . Click *Unlink provider*
> 

Reviewer: ```suggestion
.  In the organization settings, go to the *Identity providers* tab.
. Locate the *identity provider* in the list.
.+
You can use the search capabilities for this step.
. Click the action button (three dots) at the end of the line.
. Click *Unlink provider*.

```

Domain::
The domain from the organization that you want to link with the identity provider.

Hide on login page::
If this identity provider should be hidden in login pages when the user is authenticating in the scope of the organization.

Redirect when email domain matches::
If members should be automatically redirected to the identity provider when their email domain matches the domain set to the identity provider.

Once linked to an organization, the identity provider can be managed just like any other in a realm by accessing the *Identity Providers* section in the menu. However, the options herein described are only available when managing the identity provider in the scope of an organization. The only exception is the
 *Hide on login page* option that is present here for convenience.

== Editing a linked identity provider

You can edit any of the organization-related settings of a linked identity provider at any time. 
.Procedure

> . In the organization settings, go to the *Identity providers* tab.

Reviewer: ```suggestion
. In the menu, click *Organizations* and go to the *Identity providers* tab.
```

. Locate the *identity provider* in the list.
+
You can use the search option for this step.
. Click the action button (three dots) at the end of the line.
. Click *Edit*.
. Make the necessary changes.
. Click *Save*.

.Editing linked identity provider
image:images/organizations-edit-identity-provider.png[alt="Editing linked identity provider"]

== Unlinking an identity provider from an organization

When an identity provider is unlinked from an organization, it remains available as a realm-level provider that is no longer ssociated with an organization. To delete the unlinked provider, use the *Identity Providers* section in the menu.

.Procedure

> .  In the organization settings, go to the *Identity providers* tab.

Reviewer: ```suggestion
. In the menu, click *Organizations* and go to the *Identity providers* tab.
```

. Locate the *identity provider* in the list.

> .+

Reviewer: ```suggestion
+
```

Reviewer: I added the period by mistake.
